conf.yaml

```diff
@@ -5,6 +5,7 @@ meta:
 repository:
     owner: galaxyproject
     name: galaxy
+    next_milestone: "16.04"
```

Comment:
Still... I have the release process for planemo, pulsar, and galaxy-lib down to just ``make release`` - it literally figures out next versions, etc... - it would be good to get to that point with Galaxy as well.